docs/notes/previous_versions/v3.1.0.rst

```diff
@@ -0,0 +1,47 @@
+`Version 3.1.0 (latest) <https://fast-dds.docs.eprosima.com/en/v3.1.0/index.html>`_
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This minor release includes the following **features** in an ABI compatible manner:
+
+#. Allow runing CI on external contributions
+#. Allow for all durability configurations
+#. Complete support for ``Dynamic Network Interfaces``
+#. Add QoS getters from raw ``XML``
+
+This release includes the following **improvements**:
+
+#. Be less strict with parameter lengths
+#. Modify help command of ``XML CLI tool``
+#. Register and propagate ``MonitorService`` status type object
+#. Add tests for ``TypeLookup service``
+#. Convert ``Fast DDS v2`` to ``v3`` Migration Changes into a Step-by-Step User Guide
```

Comment:
I would add a link in the migration guide section and add a reference to the section here
```suggestion
#. Convert ``Fast DDS v2`` to ``v3`` Migration Changes into a :ref:`Step-by-Step User Guide <migration_guide>`.
```